### DISCUSSION TITLE
Error After Pushing Foundry project

### DISCUSSION THREAD

**Discussion by kartik01112004** [Cyfrin/foundry-full-course-cu#2585] [General]
![Screenshot 2024-09-30 205135](https://github.com/user-attachments/assets/e40907ba-a07c-4735-93c9-4234441d7786)
![Screenshot 2024-09-30 205127](https://github.com/user-attachments/assets/3f929902-6c95-4e9a-bcb6-d3203215e771)

can anyone explain

**Comment by EngrPips** [comment]
Why is the process failing at the `forge fmt` point? Please run `forge fmt` in your working directory and see if the codes format properly without error

**Reply by kartik01112004** [reply]
okay

**Comment by kartik01112004** [comment]
resolved

**Reply by EngrPips** [reply]
I am glad you figured it out. Keep crushing it.
